### ISSUE TITLE
Use of the term "compactification" on pi-Base

### ISSUE THREAD

**Issue by StevenClontz** [pi-base/data#1174]
Extending discussion coming out of #1159.

**Comment by StevenClontz** [comment]
I think I'm in camp (1.i). I've never assumed "Hausdorff" in my definition of "compact", nor "compactification"; I don't think I'm alone among working mathematicians. It seems more in the spirit of pi-Base to define things minimally, particularly with regards to separation axioms. Since there are many ways to "compactify" a space, being careful to always specify "Alexandroff [one-point] compactification" seems the most careful option.

**Comment by prabau** [comment]
That seems very reasonable and workable to me.

That matches the general definition of compactification in https://en.wikipedia.org/wiki/Compactification_(mathematics).
And as @GeoffreySangston mentioned in https://github.com/pi-base/data/pull/1159#issuecomment-2565719873, this is also backed up by some references in the literature.

**Comment by Moniker1998** [comment]
If we are going to make the change to 1.i, then we need to change all the names "the one-point compactification" to "Alexandroff one-point compactification" for all the Hausdorff spaces where the former is present. The reason being that uniqueness, even when the space is locally compact Hausdorff, is still not present. 

As for the definition, this is a minor point, but I would specify that compactification is at its core really a dense embedding $i:X\hookrightarrow Y$ into a compact space $Y$. While $Y$ is referred to as compactification, this is really the pair $(Y, i)$ that is the compactification. Of course I realize that everyone probably knows that here, but it's still an important detail that can be easily forgotten.

So we are really not dealing with "spaces" as much as with "embeddings" and arguing if those embeddings should be into Hausdorff space or not necessarily, and what to name them in either case.

For me it's two different concepts, and I'd adopt either of them depending on my needs. I don't see a point in restricting myself to just one definition, but pi-base needs to be consistent. 

> It seems more in the spirit of pi-Base to define things minimally, particularly with regards to separation axioms.

It might be true for properties which one adds to pi-base, not necessarily for concepts that we use. And I think that's an important distinction. So this doesn't convince me. 

Either way, I don't know of any ways in which non-Hausdorff compactifications are useful. So far I am not convinced that they are, no one has provided any solid reason to actually go for that definition, as far as I can see.

Could either of you @StevenClontz @prabau @GeoffreySangston provide reasons as for why those non-Hausdorff compactifications might be useful? And perhaps, why they might be useful for pi-base?

**Comment by Moniker1998** [comment]
> Question: If one starts with a Hausdorff space that is not locally compact, its "Alexandrov extension" is not Hausdorff. Would you also describe this case as a "one-point compactification" ?

I think, even though that there can be many (non-Hausdorff) one-point compactifications on a space, the Alexandrov construction is still the most famous one, so while I would change the descriptions in any case, I'd still keep the names of those spaces.

Either call "the one-point compactification" for Hausdorff compactifications (so description of S29 [One Point Compactification of the Rationals](https://topology.pi-base.org/spaces/S000029) would still need change) and Alexandrov one-point compactification/Alexandrov extension for non-Hausdorff compactifications, or be consist and go with single "Alexandrov one-point compactification" in all cases.

**Comment by StevenClontz** [comment]
> Could either of you @StevenClontz @prabau @GeoffreySangston provide reasons as for why those non-Hausdorff compactifications might be useful? And perhaps, why they might be useful for pi-base?

For one, in https://math.stackexchange.com/a/5009157/86887 @david20000813 used the trivial one-point compactification (adding a focal point whose only neighborhood is the entire space) to produce an example of a [weakly locally] compact and semiregular space which is not Baire.

**Comment by GeoffreySangston** [comment]
> Yes, it is true that we should then change many places to "Alexandroff one-point compactification". At least in the descriptions. ... But maybe you have a point that it would be too much?

There are only 15 files appearing under 'point compacti' when using the search feature on github_dev. Changing these to 'Alexandroff one-point compactification' would realistically take 5 minutes even without tools, so it doesn't seem worth worrying about. (It would also take 5 mins for the reviewer to double check no mistakes were made. But this is not worth worrying about.)

> @GeoffreySangston provide reasons as for why those non-Hausdorff compactifications might be useful? And perhaps, why they might be useful for pi-base?

It's not clear to me if this is a question about terminology or about if these appear in legitimate mathematics. I really am agnostic on this, but a contrived answer to the first is that it appears in the name of the 'Wallman–Shanin Compactification', which applies to $T_1$ spaces and could appear in pi-base one day. As you say, context could be applied to allow either usage.

If this question is about the utility of these in mathematics, I won't make any great claims here but I'll just share some things I've learned recently. The non-Hausdorff cone construction (from the finite topological spaces literature, e.g. Barmak or Peter May) = open extension (from Steen-Seebach) = extension by a (closed) focal point results in a compact space. The non-Hausdorff cone is used in Barmak's book in a way similar to how a cone is used in conventional topology (there's also a non-Hausdorff suspension which is related to the non-Hausdorff cone in the same way the conventional suspension is related to the the conventional cone), and I believe that can be used to give a succinct construction of https://en.wikipedia.org/wiki/Pseudocircle and analogs of higher dimensional spheres). An expert in finite topology would have to be consulted to give you a real answer though. The open extension is known to be the minimal 'compactification' by one point, and the Alexandroff one-point compactification / extension is known to be the maximal.

My research is related to group actions.  $\mathbb{Z}$ acts by homotheties on $\mathbb{R}^2$. The orbit space of $\mathbb{R}^2 \backslash \\{0\\}$ is called a Hopf torus, which is a basic example in flat affine geometry. The orbit space of of $\mathbb{R}^2$ is an extension of $T^2$ by a focal point. I don't know if this is useful or not, but it helps me out at least by providing at least some kind of name and information, since usually this is just the kind of thing someone would call pathological and not examine further. This kind of pathology seems to occur all of the time in geometry, e.g., the leaf space of the Reeb foliation of $S^3$ is the extension of $S^1 \sqcup S^1$ by a focal point.

**Comment by Moniker1998** [comment]
@GeoffreySangston "Wallman-Shanin compactification" that I found in Encyclopedia of General Topology seems to always be Hausdorff though, even though the construction is a bit more general as they mention.

**Comment by prabau** [comment]
> > Yes, it is true that we should then change many places to "Alexandroff one-point compactification". At least in the descriptions. ... But maybe you have a point that it would be too much?
> 
> There are only 15 files appearing under 'point compacti' when using the search feature on github_dev. Changing these to 'Alexandroff one-point compactification' would realistically take 5 minutes even without tools, so it doesn't seem worth worrying about. (It would also take 5 mins for the reviewer to double check no mistakes were made. But this is not worth worrying about.)

I agree it's not much work to do it.  When I was referring to "too much", I meant too much for readers to see, when the simpler "one-point compactification" would be adequate.

**Comment by prabau** [comment]
@Moniker1998
Forgetting about the one-point case for a moment.
Do you at least agree that the general term "compactification" can be applied the general case of a space embedded as a dense subspace of a compact space?  (that is, without any Hausdorff assumption)

That is a well attested usage, and mentioned explicitly in Encyclopedia of General Topology for example, and other sources.

**Comment by Moniker1998** [comment]
> Do you at least agree that the general term "compactification" can be applied the general case of a space embedded as a dense subspace of a compact space? (that is, without any Hausdorff assumption)

Yeah, of course. 

It does make phrasing of certain theorems awkward, like "a space is Tychonoff if and only if it admits a (Hausdorff) compactification" but even if, that's not that much of a problem, and certainly not on pi-base.

**Comment by Moniker1998** [comment]
So have we all agreed on the usage of the word compactification and can unblock #1176 ?
